src/pages/product-details/comments/comments.js

```diff
@@ -83,20 +86,16 @@ const Comments = ({ productId }) => {
 
   const rateTip = useMemo(() => handleRateTip(userId, language), [language, userId]);
 
-  const commentsLength = comments?.length;
-
-  const commentsList = comments
-    ? comments.map(({ _id, ...rest }) => (
-      <CommentsItem key={_id} data={rest} commentId={_id} productId={productId} />
-    ))
-    : [];
+  const commentsLength = currentLimit;
+  const commentsCount = comments.count;
 
-  const limitOption = commentsList?.length === comments?.length && comments?.length > commentsCount;
+  const commentsList = comments.items.map(({ _id, ...rest }) => (
+    <CommentsItem key={_id} data={rest} commentId={_id} productId={productId} />
+  ));
+  const limitOption = commentsList.length === comments.count && comments.count > commentsCount;
```

Comment:
Can you explain what is the purpose of this and how it works?

Reply:
Ok, i'll try to simplify this formula